src/test/java/org/jabref/logic/cleanup/FieldFormatterCleanupTest.java

```diff
@@ -39,7 +39,7 @@ public void setUp() {
     }
 
     @Test
-    public void testInternalAllField() throws Exception {
+    public void testInternalAllField() {
```

Comment:
Can you please leave the `throws Exception`. It's good practice to have them so that adding an exception to the method under test doesn't need a change of the test.

Reply:
Just IntelliJ complained that no exception is thrown and thus the code looked not well maintained. I'll undo it.